lightning/src/chain/keysinterface.rs

```diff
@@ -122,57 +121,69 @@ impl_writeable_tlv_based!(StaticPaymentOutputDescriptor, {
-/// claim at any point in the future) an event is generated which you must track and be able to
-/// spend on-chain. The information needed to do this is provided in this enum, including the
-/// outpoint describing which txid and output index is available, the full output which exists at
-/// that txid/index, and any keys or other information required to sign.
+/// Describes the necessary information to spend a spendable output.
+///
+/// When on-chain outputs are created by LDK (which our counterparty is not able to claim at any
+/// point in the future) a [`SpendableOutputs`] event is generated which you must track and be able
+/// to spend on-chain. The information needed to do this is provided in this enum, including the
+/// outpoint describing which `txid` and output `index` is available, the full output which exists
+/// at that `txid`/`index`, and any keys or other information required to sign.
+///
+/// [`SpendableOutputs`]: crate::util::events::Event::SpendableOutputs
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum SpendableOutputDescriptor {
-	/// An output to a script which was provided via KeysInterface directly, either from
-	/// `get_destination_script()` or `get_shutdown_scriptpubkey()`, thus you should already know
-	/// how to spend it. No secret keys are provided as rust-lightning was never given any key.
+	/// An output to a script which was provided via [`KeysInterface`] directly, either from
+	/// [`get_destination_script`] or [`get_shutdown_scriptpubkey`], thus you should already
+	/// know how to spend it. No secret keys are provided as LDK was never given any key.
 	/// These may include outputs from a transaction punishing our counterparty or claiming an HTLC
 	/// on-chain using the payment preimage or after it has timed out.
+	///
+	/// [`get_shutdown_scriptpubkey`]: KeysInterface::get_shutdown_scriptpubkey
+	/// [`get_destination_script`]: KeysInterface::get_shutdown_scriptpubkey
 	StaticOutput {
-		/// The outpoint which is spendable
+		/// The outpoint which is spendable.
 		outpoint: OutPoint,
 		/// The output which is referenced by the given outpoint.
 		output: TxOut,
 	},
-	/// An output to a P2WSH script which can be spent with a single signature after a CSV delay.
+	/// An output to a `P2WSH` script which can be spent with a single signature after an `OP_CSV`
+	/// delay.
 	///
 	/// The witness in the spending input should be:
+	/// ```bitcoin
 	/// <BIP 143 signature> <empty vector> (MINIMALIF standard rule) <provided witnessScript>
+	/// ```
 	///
-	/// Note that the nSequence field in the spending input must be set to to_self_delay
-	/// (which means the transaction is not broadcastable until at least to_self_delay
-	/// blocks after the outpoint confirms).
+	/// Note that the `nSequence` field in the spending input must be set to `to_self_delay` (which
+	/// means the transaction is not broadcastable until at least `to_self_delay` blocks after the
+	/// outpoint confirms).
```

Comment:
You can add a reference to BIP68 here: https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki for further explanation on the relative-timelock semantic.

Reply:
Done.